scripts/globals/items.lua

```diff
@@ -640,7 +640,18 @@ xi.items =
     MOCKING_BEAK                    = 3092,
     SPOTTED_FLYFROND                = 3096,
     BLACK_RABBIT_TAIL               = 3097,
+    GNR_TAURUS_HORN                 = 3098,
```

Comment:
For items.lua entries, please use the uppercased value used in the `name` field of the db tables.  So this, for example would correspond to:
`INSERT INTO `item_basic` VALUES (3098,0,'gnarled_taurus_horn','gnr._taurus_horn',1,36868,46,1,0);`

and become GNARLED_TAURUS_HORN

Exceptions to the rule: `-` will be come `_`, `+1` would become `P1`, and `-1` would become `M1`